ci-operator/step-registry/aws/provision/localzone/aws-provision-localzone-commands.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
```

Comment:
should be adapted with `ci-operator/step-registry/aws/provision/localzone-subnets/aws-provision-localzone-subnets-commands.sh`